client/src/pages/Landing.jsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from "react";
+import "../styles/Landing.css";
+import { Button, Typography } from "@material-ui/core";
+
+function LandingPage() {
+  
+  const [start, setStart] = useState(true);
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [topMessage, setTopMessage] = useState("Welcome back!");
+  const [bottomMessage, setBottomMessage] = useState("Don't have an account?");
+  const [logSign, setLogSign] = useState("Login");
+  const [createLogin, setCreateLogin] = useState("Create");
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    console.log(email, password);
+    // API verification steps ...
+  };
+
+  const handleLowerButton = (e) => {
+    if (start) {
+      setTopMessage("Sign up to Kanban");
+      setLogSign("Sign up");
+      setBottomMessage("Already have an account?");
+      setCreateLogin("Login");
+      setStart(false);
+    } else {
+      setTopMessage("Welcome back!");
+      setLogSign("Login");
+      setBottomMessage("Don't have an account?");
+      setCreateLogin("Create");
+      setStart(true);
+    }
+  };
+
+  return (
+    <div className="container-main">
+      <div className="container-content">
```

Comment:
Try to get the content to fill the page rather than surrounding it with a border